metricbeat/module/apache/status/status_test.go

```diff
@@ -15,8 +15,6 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// +build !integration
```

Comment:
Why was this removed?

Reply:
Sorry, I usually have to remove it so Goland is able to work on that file. I forgot to undo it. Thanks!